cybox/utils/__init__.py

```diff
@@ -73,6 +74,17 @@ def _import_submodules(pkg):
         importlib.import_module(mod_name)
 
 
+def is_sequence(item):
```

Comment:
I'd be totally fine with removing the `Mapping` clause.

Originally, I felt like the behavior when passing in a dict would be kinda weird (though totally consistent with other builtin methods that take iterables as parameters) so I restricted it. The more I think about it, though more I think removing that restriction _might_ be the best option, since it's pretty consistent with other Python functions.

I think I originally wrote this for python-stix because I use tuples and sets a lot and got frustrated by the `list` requirement after seeing one-too-many stack traces :)